src/components/Breadcrumb/Breadcrumb.vue

```diff
@@ -0,0 +1,148 @@
+<!--
+ - @copyright Copyright (c) 2020 Raimund Schlüßler <raimund.schluessler@mailbox.org>
+ -
+ - @author Raimund Schlüßler <raimund.schluessler@mailbox.org>
+ -
+ - @license GNU AGPL version 3 or any later version
+ -
+ - This program is free software: you can redistribute it and/or modify
+ - it under the terms of the GNU Affero General Public License as
+ - published by the Free Software Foundation, either version 3 of the
+ - License, or (at your option) any later version.
+ -
+ - This program is distributed in the hope that it will be useful,
+ - but WITHOUT ANY WARRANTY; without even the implied warranty of
+ - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ - GNU Affero General Public License for more details.
+ -
+ - You should have received a copy of the GNU Affero General Public License
+ - along with this program. If not, see <http://www.gnu.org/licenses/>.
+ -
+ -->
+
+<docs>
+
+### General description
+
+This component is meant to be used inside a Breadcrumbs component.
+
+</docs>
+
+<template>
+	<div class="crumb"
+		ref="crumb"
+		:class="{over: hovering}"
+		draggable="false"
+		@dragstart="dragstart"
+		@drop="dropped($event)"
+		@dragover="dragOver($event)"
+		@dragenter="($event) => dragEnter($event)"
+		@dragleave="dragLeave">
+		<element :is="tag" :to="to" :href="href">
+			<span v-if="icon" :class="icon" class="icon" />
+			<span v-else>{{ title }}</span>
+		</element>
+	</div>
+</template>
+
+<script>
+export default {
+	name: 'Breadcrumb',
+	props: {
+		/**
+		 * The displayed title of the breadcrumb.
+		 */
+		title: {
+			type: String,
+			required: true,
+		},
+		/**
+		 * The router-link to prop [https://router.vuejs.org/api/#to](https://router.vuejs.org/api/#to)
+		 * If set, the breadcrumbs will be rendered by router-link.
+		 */
+		to: {
+			type: String,
+			required: false,
+			default: undefined,
+		},
+		/**
+		 * Set this prop if your app doesn't use vue-router, breadcrumbs will show as normal links.
+		 */
+		href: {
+			type: String,
+			required: false,
+			default: undefined,
+		},
+		/**
+		 * Set a css icon-class to show an icon instead of the title text.
+		 */
+		icon: {
+			type: String,
+			required: false,
+			default: '',
+		},
+	},
+	data: function() {
+		return {
+			hovering: false,
+		}
+	},
+	computed: {
+		/**
+		 * determines which element tag to use
+		 * @returns {String} the tag
+		 */
+		tag() {
+			return this.to ? 'router-link' : 'a'
+		},
+	},
+	methods: {
+		cancelEvent(e) {
+			e.stopPropagation()
+			e.preventDefault()
+			return false
+		},
+		dragstart(e) {
+			return this.cancelEvent(e)
+		},
+		dropped(e) {
+			this.cancelEvent(e)
+			this.$parent.$emit('dropped', this.to || this.href)
```

Comment:
> 
> 
> Please also emit for this component maybe?
> So we can chose between watching a single one versus making all of the crumbs droppable?

Fine with me. But one has to be careful then. If you have handlers on both, you will also get two events fired.

Reply:
> Also, could be nice if we could disable the droppable function on a per-crumb basis?
> What do you think? Too much?

Makes sense. Dropping on the last crumb might be unwanted, since the item is already in this folder. Also helpful if moving something to this folder is not allowed (read-only?). I will add that.

Reply:
Dropping can be disabled now by `:disableDrop`.